blog/2022-03-02-os.mdx

```diff
@@ -10,4 +10,4 @@ We are excited to announce that IaSQL is now open source. The main repository is
 
 PostgreSQL IaSQL databases can be provisioned and configured via the [CLI](https://github.com/iasql/cli) which is written in Rust. The CLI calls the [engine](https://github.com/iasql/iasql-engine) which is a Node.js server written in Typescript that provisions unmodified PostgreSQL instances loaded with tables representing AWS services controlled via the [AWS SDK](https://www.npmjs.com/package/aws-sdk). AWS is our main focus at the moment, but we plan to support GCP, Azure and other cloud providers soon. This is an [updated list](https://github.com/iasql/iasql-engine#cloud-providers) of the AWS services that we currently support. Let us know if you need a specific AWS service and we should be able prioritize it!
 
-We want to make it easier for community members to write IaSQL modules, easier to use IaSQL with migration systems, add more supporting documentation, reduce waiting times when provisioning infrastructure, add more functionality to the existing AWS services and simplify some of the existing tables. The list of things we want to build into IaSQL is long, but we want to do it in the open with your feedback and help. Drop us a line on [Discord](https://discord.com/invite/machGGczea)!
\ No newline at end of file
+We want to make it easier to write IaSQL modules, reduce waiting times when provisioning infrastructure and add more functionality to the existing AWS services. The list of things we want to build into IaSQL is long, but we want to do it in the open with your feedback and help. Drop us a line on [Discord](https://discord.com/invite/machGGczea)!
```

Comment:
Maybe:

```suggestion
We want to make it easier to write IaSQL modules, reduce waiting times when provisioning infrastructure, add more functionality to the existing AWS services, and so on. The list of things we want to build into IaSQL is long, but we want to do it in the open with your feedback and help. Drop us a line on [Discord](https://discord.com/invite/machGGczea)!
```

Now it's a list of three but explicitly saying that the real list is longer so the next sentence doesn't contradict the first (technically)